examples/Bunch/include/simulation_defines/param/physicalConstants.param

```diff
@@ -26,6 +26,11 @@ namespace picongpu
 {
     const float_64 PI = 3.141592653589793238462643383279502884197169399;
 
+    /** Discriminant used for calculations that want to separate between
```

Comment:
Great idea to have this as new parameter. :+1: 

Please use `Limit`instead of [`Discriminant`](http://en.wikipedia.org/wiki/Discriminant).